packages/react/src/SubNav/SubNav.module.css

```
@@ -459,6 +459,14 @@
   justify-content: flex-end;
 }
 
+.SubNav__link:focus-visible,
+.SubNav__heading:focus-visible,
+.SubNav__sub-menu-toggle:focus-visible,
+.SubNav__overlay-toggle:focus-visible {
+  outline: var(--brand-borderWidth-thick) solid var(--brand-color-focus);
+  outline-offset: 2px;
```

2px is available on our base scale. Let's use that variable going forward...

```suggestion
  outline-offset: var(--base-size-2);
```